### DISCUSSION TITLE
How to cancel the styles of h1~h6 and prohibit modification

### DISCUSSION THREAD

**Discussion by bbhxwl** [tailwindlabs/tailwindcss#13005] [Help]
How to cancel the styles of h1~h6 and prohibit modification?

![image](https://github.com/tailwindlabs/tailwindcss/assets/26525599/8da2e0d9-b72f-4c02-9e0b-6ab13b0529a0)

**Comment by wongjn** [comment]
I don't think you could prohibit modification, but you can remove the reset styles of the `h1`-`h6` by [disabling preflight](https://tailwindcss.com/docs/preflight#disabling-preflight).

**Reply by bbhxwl** [reply]
I just want to disable h1 to h6 because they will be used in markup and cannot set font size. Is there any better way?

**Reply by wongjn** [reply]
Yes, you could consider using the [`@tailwindcss/typography` plugin](https://github.com/tailwindlabs/tailwindcss-typography).
